consumer/pdata/log.go

```diff
@@ -29,56 +29,51 @@ import (
 // Must use NewLogs functions to create new instances.
 // Important: zero-initialized instance is not valid for use.
 type Logs struct {
-	orig *[]*otlplogs.ResourceLogs
+	wrapper internal.LogsWrapper
```

Comment:
It looks a bit strange and is not uniform with the rest of pdata structs. Not sure the change is worth it.

If someone is accessing `internal.LogsWrapper` it is on them if we break them. I don't think we need to solve it.

Reply:
Why is that not uniform? I think we need to do the same thing anyway for metrics and traces since the initial wrapper PR was only for logs.

Reply:
See https://github.com/open-telemetry/opentelemetry-collector/pull/2693 for metrics, I will do for trace the same thing.

Reply:
Everything else uses this pattern of `orig` and pointing to Protobuf-generated messages. This is using something caller Wrapper, not clear why it is different.

Reply:
ok, will change to use origin.

Reply:
Done. Are you ok with this change now?